src/include/daos_errno.h

```diff
@@ -295,6 +295,15 @@ D_FOREACH_ERR_RANGE(D_DEFINE_RANGE_ERRNO)
  */
 const char *d_errstr(int rc);
 
+/** Fetch a string associated with a registered gurt errno
```

Comment:
I think it would be helpful to make it clear that there is no memory allocation being done, and that the caller does not free any memory associated with the return string.